### ISSUE TITLE
python error when try to save results with "save all results"

### ISSUE THREAD

**Issue by gioman** [boundlessgeo/qgis-tester-plugin#4]
```
Traceback (most recent call last):
  File "C:/Users/qgis/.qgis2/python/plugins\qgistester\reportdialog.py", line 35, in 
    self.actionSaveAll.triggered.connect(lambda: self.saveResults(True))
  File "C:/Users/qgis/.qgis2/python/plugins\qgistester\reportdialog.py", line 108, in saveResults
    out += '{}'.format(results.result)
AttributeError: TestResult instance has no attribute 'result
```

"save selected results" seems to work as expected.

**Comment by gioman** [comment]
@alexbruy thanks for the fix, but seems that there is another problem


```
Traceback (most recent call last):
  File "C:/Users/qgis/.qgis2/python/plugins\qgistester\reportdialog.py", line 35, in 
    self.actionSaveAll.triggered.connect(lambda: self.saveResults(True))
  File "C:/Users/qgis/.qgis2/python/plugins\qgistester\reportdialog.py", line 123, in saveResults
    f.write(out)
  File "C:\PROGRA~1\BOUNDL~1\Desktop\osgeo4w\apps\Python27\lib\codecs.py", line 691, in write
    return self.writer.write(data)
  File "C:\PROGRA~1\BOUNDL~1\Desktop\osgeo4w\apps\Python27\lib\codecs.py", line 351, in write
    data, consumed = self.encode(object, self.errors)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd0 in position 6358: ordinal not in range(128)
```

**Comment by alexbruy** [comment]
Second issue fixed in 5f7c22091a
